www/TouchID.js

```diff
@@ -1,8 +1,8 @@
 function TouchID() {
 }
 
-TouchID.prototype.isAvailable = function (successCallback, errorCallback) {
-  cordova.exec(successCallback, errorCallback, "TouchID", "isAvailable", []);
+TouchID.prototype.isAvailable = function (checkBiometricType, successCallback, errorCallback) {
```

Comment:
Hi! Very grateful for your changes! The only thing that worries me is `isAvailable` will break for everyone because the first param was changed. Can you consider moving `checkBiometricType` to the end of the param list?